docs/content/release_notes/limitations.md

```diff
@@ -91,6 +91,12 @@ The following limitations apply when expanding volumes with the IBM block storag
 
 Volume group configuration is only supported for use with IBM Spectrum Virtualize family storage systems.
 
+In addition, volume groups can only be defined in one of the following ways:
```

Comment:
I think it is more correct to say PVCs can only be defined inside the volume groups in the following way

Reply:
updated